src/AWSCredentials.jl

```diff
@@ -638,11 +638,12 @@ end
 Determine the current AWS region that should be used for AWS requests. The order of
 precedence mirrors what is [used by the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-precedence):
 
-1. Environmental variable: as specified by the `AWS_DEFAULT_REGION` environmental variable.
-2. AWS configuration file: `region` as specified by the `profile` in the configuration
+1. Environment variable: `AWS_REGION`.
+2. Environment variable: `AWS_DEFAULT_REGION` (used if `AWS_REGION` is not set).
```

Comment:
As this list is stating the precedence ordering the extra comment is unnecessary:
```suggestion
1. Environment variable: `AWS_REGION`.
2. Environment variable: `AWS_DEFAULT_REGION`
```

Alternatively, if we think of this list as different types of region sources then we could combine "environment variables" into one list item:
```suggestion
1. Environment variables: `AWS_REGION` if set, otherwise `AWS_DEFAULT_REGION`.
```